service/history/nDCTransactionMgr.go

```diff
@@ -102,13 +104,15 @@ type (
 			ctx ctx.Context,
 			now time.Time,
 			targetWorkflow nDCWorkflow,
+			reapplyEvents *persistence.WorkflowEvents,
```

Comment:
why do we need this input parameter?

Reply:
events to be persisted to DB can be accessible from closing of a transaction. so this reapplyEvents should be removed.

Reply:
Moved everything to workflow context